src/platform/packages/shared/kbn-unified-tabs/src/types.ts

```diff
@@ -26,6 +26,25 @@ export interface TabsSizeConfig {
   // TODO: extend with possibly different sizes for pinned tabs
 }
 
+// TODO adjust interface when real data is available, this currently types TAB_CONTENT_MOCK
+export interface PreviewContentConfig {
+  id: number;
+  name: string;
+  query: {
+    language: 'esql' | 'kql';
+    query: string;
+  };
+  status: 'success' | 'running' | 'danger'; // status for now matches EuiHealth colors for mocking simplicity
```

Comment:
Would be better to define as a enum type.

Reply:
Could you explain why and if it applies to `query.language` in line 34 as well? 
No problem from my side with changing it, just want to understand the reasoning

Reply:
Using enums in these cases is more preferable so we can reference them later via `previewContent.status === TabStatus.running` in a named way instead of `previewContent.status === 'running'`.

For `query` we can use the existing `AggregateQuery` type from `@kbn/es-query`.

Reply:
Got it, thank you! I was more focused on drawbacks of Enum, but if it's the preferred way in Kibana and/or our team, I have no problem with switching to it 👍